src/common/getToken.js

```diff
@@ -0,0 +1,12 @@
+import axios from 'axios';
+
+export async function getToken() {
+  try {
+    const res = await axios.get('http://localhost:3000/auth');
```

Comment:
magic string!

Reply:
I don't understand what this getToken method. Wrapper for wrapper?

Reply:
To get token from my node server and set it in local storage.